File: uri 1061.py
start_day = input().split(" ")
start_day = int(start_day[1])
start_hour, start_min, start_sec = map(int, input().split(" : "))
end_day = input().split(" ")
end_day = int(end_day[1])
end_hour, end_min, end_sec = map(int, input().split(" : "))
day = end_day - start_day
hour = end_hour - start_hour
min = end_min - start_min
sec = end_sec - start_sec

if sec < 0:
    sec += 60
    min -= 1
if min < 0:
    min += 60
    hour -= 1
if hour < 0:
    hour += 24
    day -= 1
print(f"{day} dia(s)")
print(f"{hour} hora(s)")
print(f"{min} minuto(s)")
print(f"{sec} segundo(s)")

File: uri 1061.c
#include <stdio.h>
#include <stdlib.h>

int main()
{
    int d,dd,h,hh,m,mm,s,ss;
    char k[5];
    scanf("%s %d",k,&d);
    scanf("%d %s %d %s %d",&h,k,&m,k,&s);
    scanf("%s %d",k,&dd);
    scanf("%d %s %d %s %d",&hh,k,&mm,k,&ss);

    s=ss-s;
    m=mm-m;
    h=hh-h;
    d=dd-d;

    if(s<0){
        s+=60;
        m--;
    }
    if(m<0){
        m+=60;
        h--;
    }
    if(h<0){
        h+=24;
        d--;
    }
    printf("%d dia(s)\n%d hora(s)\n%d minuto(s)\n%d segundo(s)\n",d,h,m,s);

    return 0;
}
